CHANGES

```diff
@@ -22,16 +22,148 @@ description indicates whether it is SOURCE-COMPATIBLE, BINARY-COMPATIBLE, or BRE
 Further information about [seL4 releases](https://docs.sel4.systems/sel4_release/) is available.
 
 ---
-Upcoming release: BREAKING
+Upcoming release: BINARY COMPATIBLE
 
 ## Changes
 
+## Upgrade Notes
+---
+12.0.0 2020-10-21: BREAKING
+
+## Changes
+
+* Arm SMMU v2 kernel API
 * aarch32: Moved TPIDRURO (PL0 Read-Only Thread ID register) to TCB register context from VCPU registers. This means
   changes to this register from user level have to go via seL4_TCB_Write Registers instead of seL4_ARM_VCPU_WriteRegs.
 * aarch64,cortex-a53,hyp: Reduce seL4_UserTop when on a cortex-a53 platform and KernelArmHypervisorSupport is set.
   This is because the kernel uses the last slot in the top level VSpace object for storing the assigned VMID and so
   any addresses that are addressed by the final slot are not accessible.
   This would apply to any CPU that have 40bit stage 2 translation input address.
+* Use correct configuration for TLS base.
```

Comment:
Changes listed in the release notes should only document things that are user visible - API changes or behaviour changes. Remove anything that is superfluous to this - e.g "tweak section alignment".